src/System.Drawing.Common/src/System/Drawing/Image.cs

```diff
@@ -46,7 +48,52 @@ public object Tag
             set => _userData = value;
         }
 
-        internal Image() { }
+        private protected Image() { }
+
+#pragma warning disable CA2229 // Implement Serialization constructor
```

Comment:
Do we need these? I didn't think we ran FXCop/Code analysis.

Reply:
yes, this is required, otherwise it won't compile, at least not with VS.